packages/account/src/Configs/personal-details-config.ts

```diff
@@ -85,14 +83,14 @@ export const personal_details_config = ({
         place_of_birth: {
             supported_in: ['maltainvest', 'iom', 'malta'],
             default_value: account_settings.place_of_birth
-                ? residence_list.find(item => item.value === account_settings.place_of_birth)?.text
+                ? residence_list.find(item => item.value === account_settings.place_of_birth)?.text ?? ''
                 : '',
             rules: [['req', localize('Place of birth is required.')]],
         },
         citizen: {
             supported_in: ['iom', 'malta', 'maltainvest'],
             default_value: account_settings.citizen
-                ? residence_list.find(item => item.value === account_settings.citizen)?.text
+                ? residence_list.find(item => item.value === account_settings.citizen)?.text ?? ''
```

Comment:
Same here

Reply:
yes, we are not supposed to pass undefined to default_value. TS assumes, that id could not be found.

Reply:
Then we need to do the same other default_value, doesn't look good this way
If we really need this, let's come up with a better solution